libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -835,7 +845,7 @@ button.gd-button-primary.gd-upsell-button {
     font-style: normal;
     font-weight: normal;
     font-size: 14px;
-    margin: auto 7px;
+    margin: auto 7px auto 10px;
```

Comment:
Jus a confirmation: do we really need to change this? Can we set the margin in `gd-header-countdown` instead?